src/content/learn/editor-setup.md

```diff
@@ -36,7 +36,7 @@ Code linters find problems in your code as you write, helping you fix them early
 * [Install ESLint with the recommended configuration for React](https://www.npmjs.com/package/eslint-config-react-app) (be sure you have [Node installed!](https://nodejs.org/en/download/current/))
 * [Integrate ESLint in VSCode with the official extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
 
-**Make sure that you've enabled all the [`eslint-plugin-react-hooks`](https://www.npmjs.com/package/eslint-plugin-react-hooks) rules for your project.** They are essential and catch the most severe bugs early. The recommended [`eslint-config-react-app`](https://www.npmjs.com/package/eslint-config-react-app) preset already includes them.
+**Make sure that you've enabled all the [`eslint-plugin-react-hooks`](https://www.npmjs.com/package/eslint-plugin-react-hooks) rules in your project.** They are essential and catch the most severe bugs early. If you’re setting up linting manually, we recommend using both [`eslint-plugin-react`](https://www.npmjs.com/package/eslint-plugin-react) and [`eslint-plugin-react-hooks`](https://www.npmjs.com/package/eslint-plugin-react-hooks). If you’re using the new ESLint flat config format, you can extend the configs included with those plugins.
```

Comment:
See my comment in the issue - I don't think we should recommend one or the other. The goal here is to inform users that if they are using one of the config presets, it's already included.